File: ejercicios/python/ejercicio083.py
import random

lista = []
for i in range(10):
    lista.append(random.randint(100, 200))

print("Elementos del arreglo original: ")
print(lista)

for i in range(10):
    for j in range(i,10):
        if lista[i] > lista[j]:
            temp = lista[i]
            lista[i] = lista[j]
            lista[j] = temp
print()

print("Elementos del arreglo ordenado: ")
print(lista)

File: ejercicios/c++/ejercicio083.cpp
#include <iostream>
#include <ctime>
#include <cstdlib>
using namespace std;

int main(){
	int ar[10];
	int temp;
	srand(time(0));
	
	for(int i = 0; i < 10; i++){
		ar[i] = (rand() % 100) + 100;
	}
	
	cout << "Elementos del arreglo original: " << endl;
	for(int i=0;i<10;i++)
		cout << ar[i] << "  "; 
	
	cout << endl;
	
	for(int i = 0; i < 10; i++){
		for(int j = i; j < 10; j++){
			if(ar[i] > ar[j]){
				temp = ar[i];
				ar[i] = ar[j];
				ar[j] = temp;
			}
		}
	}
	
	cout << "Elementos del arreglo ordenado: " << endl;
	for(int i=0;i<10;i++)
		cout << ar[i] << "  "; 
	
	cout << endl;
	
	return 0;
}
